pyproject.toml

```diff
@@ -25,6 +30,10 @@ classifiers = [
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
We could automate that with https://pyproject-fmt.readthedocs.io/en/latest/#calculating-max-supported-python-version. Though I think trove classifiers for Python version has been deprecated in favor of `requires-python` pyproject.toml field. In any case, we should also adopt `requires-python`.